components/ide/code/leeway.Dockerfile

```diff
@@ -45,6 +46,19 @@ RUN yarn --frozen-lockfile --network-timeout 180000
 RUN rm -rf remote/node_modules/
 COPY --from=dependencies_builder /gp-code/remote/node_modules/ /gp-code/remote/node_modules/
 
+RUN tmp1=$(mktemp) && \
+    tmp2=$(mktemp) && \
+    tmp3=$(mktemp) && \
+    name=$(jq --raw-output '.nameShort' product.json) && \
+    if [ "$CODE_QUALITY" = "insider" ]; then \
+        name="$name - Insiders" \
+    ; fi  && \
+    jq --arg codeQuality "$CODE_QUALITY" '. += {"quality":$codeQuality}' product.json > "$tmp1" && \
+    jq --arg name "$name" '.nameShort=$name' "$tmp1" > "$tmp2" && \
+    jq --arg name "$name" '.nameLong=$name' "$tmp2" > "$tmp3" && \
+    rm product.json && \
+    mv "$tmp3" product.json
```

Comment:
Here's an [example for applying changes in one line](https://github.com/VSCodium/vscodium/blob/e480df5b1f0039321c0953b621f7ef085bae9c91/prepare_vscode.sh#L77-L78) :smile: